src/main/java/com/blamejared/crafttweaker/impl/fluid/MCFluidStack.java

```diff
@@ -59,4 +59,21 @@ public MCFluid getFluid() {
     public FluidStack getInternal() {
         return stack;
     }
+    
+    @Override
+    public boolean equals(Object o) {
+        if(this == o)
+            return true;
+        if(o == null || getClass() != o.getClass())
+            return false;
+        
+        MCFluidStack that = (MCFluidStack) o;
+    
+        return stack.equals(that.stack);
```

Comment:
Is it intended to not compare the size of the stack here? (FluidStack doesn't take size into account normally for equals, though it does for hashCode)